examples/vadd/Makefile

```diff
@@ -0,0 +1,72 @@
+F90 ?= gfortran
+
+
+KERNELS = vadddevice.cl
```

Comment:
Please add a comment to say that this line is only used when kernels are built at compile time instead of JIT.

Reply:
Done